README.md

```diff
@@ -1,13 +1,13 @@
-WebExtensions / Chrome browser add-on for linking a browser to the KeePassRPC KeePass plugin.
+WebExtensions (Mozilla Firefox) / Google Chrome (Chromium) browser add-on for linking browser to KeePassRPC KeePass plugin.
 
 Build requirements:
 
-# node
-# npm
-# the usual hundreds of node modules installed by the above
+* node
+* npm or yarn
```

Comment:
I have tested it and it looks interoperable. If anyone finds the problem, removing it is simple.